src/components/Title/index.js

```diff
@@ -4,10 +4,9 @@ import { Text, View } from 'react-native';
 import styles from './styles';
 
 const Title = () => {
-  const { textStyle, viewStyle } = styles;
   return (
-    <View style={viewStyle}>
-      <Text style={textStyle}>Todo List</Text>
+    <View style={styles.view}>
+      <Text style={styles.text}>Todo List</Text>
```

Comment:
Move this string to a constant.

Reply:
I've moved both screens to "src/screens" on a new branch called "bookList".
About the constant requested, it's now managed by router's "navigationOption" configuration.